apps-rendering/src/fixtures/live.ts

```diff
@@ -260,7 +260,7 @@ const fields = {
 	webUrl: '',
 };
 
-const live: Liveblog = {
+const live: LiveBlog = {
 	design: Design.LiveBlog,
 	...fields,
 	blocks: [],
```

Comment:
Is it worth adding blocks data here, so that we can see the key events in the storybook for live blog?

Reply:
Yes, good idea. I'll do that in a follow-up.